_posts/2020-08-31-improving-decompiler-widget-gsoc.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Improving Decompiler Widget - GSoC Project"
+date: 2020-05-20
+tags: ["Decompiler", "GSoC"]
+categories: ["Cutter", "Decompiler", "GSoC"]
+author: nirmal_manoj
+# post_image: "/assets/images/blog/posts/improving-decompiler-widget-gsoc/decompiler_1.png"
```

Comment:
what about this?

Reply:
@ITAYC0HEN Please suggest some choices for the post image.

Reply:
decompiled code :)